src/Communication/Socket/MockSocket.php

```diff
@@ -124,4 +124,11 @@ public function disconnect($reason = 1000)
 
         return true;
     }
+
+    public function waitForData(float $maxSeconds): bool
+    {
+        // TODO: Implement wait if needed.
+
+        return !empty($this->receivedData);
```

Comment:
please don't use the empty function

Reply:
I have deleted implementation of `waitForData()` from `MockSocket` because it is not used and not needed to implement the interface.